requirements.txt

```diff
@@ -0,0 +1 @@
+sympy
```

Comment:
Is there a specific syntax for these files, or are we just writing them as free-form text files?

Reply:
Part of the [PIP specification](https://pip.readthedocs.io/en/1.1/requirements.html)